index.html

```diff
@@ -13,7 +13,9 @@
         <h2>Vertical Navigation Structure Examples</h2>
         <ul>
           <li><a href="nav.html">Navbar Latest and greatest</a></li>
+          <li><a href="describedby-submenu.html">Navbar as list that uses aria-describedby on submenu links</a></li>
```

Comment:
Given my comment above, what are your thoughts on removing this as a navigation example, and creating a static test page with different techniques for using aria-label, aria-labelledby and aria-describedby?